### PR TITLE
feat: port the "one empty line" and "whitespace" rules to `wdl-lint`.

### PR DESCRIPTION
**Pull request by peterhuene**
This commit combines the "one empty line" and "whitespace" rules as the new "whitespace" rule in `wdl-lint`.

Before submitting this PR, please make sure:

- [x] You have added a few sentences describing the PR here.
- [x] You have added yourself or the appropriate individual as the assignee.
- [x] You have added at least one relevant code reviewer to the PR.
- [x] Your code builds clean without any errors or warnings.
- [x] You have added tests (when appropriate).
- [x] You have updated the README or other documentation to account for these
      changes (when appropriate).
- [x] You have added an entry to the relevant `CHANGELOG.md` (see
      ["keep a changelog"] for more information).
- [x] Your commit messages follow the [conventional commit] style.

[conventional commit]: https://www.conventionalcommits.org/en/v1.0.0/#summary
["keep a changelog"]: https://keepachangelog.com/en/1.0.0/

### CODE REVIEW

**Review by a-frantz — APPROVED** [review]
Nice work. Just a small thing then we can merge

**Comment by peterhuene** [comment]
@a-frantz I think it left out your review comment for me to address.

File: wdl-lint/tests/lints/only-whitespace/source.errors [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+warning: line contains only whitespace [rule: Whitespace]
+   ┌─ tests/lints/only-whitespace/source.wdl:16:1
+   │
+16 │      
+   │ ^^^^^
+   │
+   = fix: remove the whitespace from this line
+
+warning: missing newline at the end of the file [rule: EndingNewline]
```
Comment (a-frantz) [inline wdl-lint/tests/lints/only-whitespace/source.errors:60]: Oi vey. Let's see if this goes through. It seems like a mistake to have this warning in the Whitespace test case.

Comment (peterhuene) [reply wdl-lint/tests/lints/only-whitespace/source.errors:60]: I can remove it.

**Review by a-frantz — APPROVED** [review]

**Review by adthrasher — APPROVED** [review]

**Pull request merged**